test-apps/appui-test-app/appui-test-providers/src/Translation.ts

```diff
@@ -0,0 +1,22 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
Should we rename the file to `useTranslation`?

Reply:
I thought about it but was 50/50 on it. Since you mentioned it, I decided to rename it.